src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.js

```diff
@@ -1627,6 +1644,24 @@ export default defineComponent({
       shakaOverflowMenu.registerElement('ft_audio_tracks', new AudioTrackSelectionFactory())
     }
 
+    function registerAutoplayToggle() {
```

Comment:
As it's possible that the user could adjust the auto play settings in another window while this player is open, I would suggest adding a watcher to `props.autoplayEnabled` and notifying the button about the change with an event, so that it doesn't get out of sync.

Reply:
Isn't this being done by us passing `autoplayEnabled` as a prop from the `Watch.js` `autoplayEnabled` computed property?

Reply:
That just for passing the initial value to the button, it passes a raw boolean. shaka-player and the custom player elements are outside of Vue, so they don't have reactivity, which means we have to take care of reacting to changes ourselves.